src/commons/LDDT.h

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <unordered_map>
+#include <cmath>
+#include <map>
+
+#ifndef LDDT_H
+#define LDDT_H
+
+#define INF 3.40282e+038 // TODO: More elegant way of handling divide-by-zero
+const float cutoff = 15.0;
+
+class LDDTcalculator {
+public:
+    LDDTcalculator() {}
+    LDDTcalculator(int qLen, int tLen) {
+        queryLength = qLen, targetLength = tLen;
```

Comment:
I would avoid using this construct.
It is better to write:
```
queryLength = qLen;
targetLength = tLen;
```
but even better would be
```
LDDTcalculator(int qLen, int tLen) : queryLength(qLen), targetLength(tLen){
}
```